scanner/mappers/mappers.go

```diff
@@ -325,10 +325,17 @@ func toProtoV4VulnerabilitiesMap(
 				Vector:    sev.v3Vector,
 			}
 		}
+		// Uses NVD description if not provided.
+		description := v.Description
+		if nvd, ok := nvdVulns[v.ID]; ok {
+			if description == "" && len(nvd.Descriptions) > 0 {
+				description = nvd.Descriptions[0].Value
```

Comment:
instead of getting the first one, let's get the one in English (`Lang == "en"`)

Reply:
The enricher already ensures that only "en" is available.